src/Castle.Core.Tests/project.json

```diff
@@ -9,22 +9,27 @@
     "Castle.Core": { "target": "project", "version" : "" },
     "Castle.Services.Logging.SerilogIntegration": { "target": "project", "version" : "" },
     "Castle.Services.Logging.NLogIntegration": { "target": "project", "version": ""},
-    "Microsoft.CSharp": "4.0.0",
     "Serilog.Sinks.TextWriter": "2.0.0-beta-505",
-    "System.Console": "4.0.0-beta-23516",
-    "System.Diagnostics.TraceSource": "4.0.0-beta-23516",
-    "xunit": "2.1.0",
-    "xunit.runner.dnx": "2.1.0-rc1-build204"
+    "xunit": "2.1.0"
   },
   "commands": {
     "test": "xunit.runner.dnx"
```

Comment:
In RC2 there will be a "testRunner" property: [an example](https://github.com/aspnet/Hosting/blob/6c1247b30cb005aec4091904ecefe8a990296ae0/test/Microsoft.AspNetCore.Hosting.Tests/project.json).